### PR TITLE
chore: Add persistentvolumeclaim-storageclass-policy repository

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Part of https://github.com/kubewarden/kubewarden-controller/issues/688

## Test

Tested with `terraform plan`.

### CODE REVIEW

File: terraform/repository_policies.tf [hunk 1]
```diff
@@ -573,3 +573,15 @@ module "policy_repository_unique_service_selector" {
     github = github.kubewarden
   }
 }
+
+module "policy_repository_persistentvolumeclaim" {
+  source = "./modules/policy_repository"
+
+  name                   = "persistentvolumeclaim-policy"
```
Comment (flavio) [inline terraform/repository_policies.tf:580]: Maybe we will provide other policies related with PVC. What about: "persistentvolumeclaim-storageclass-policy"?

Comment (jvanz) [reply terraform/repository_policies.tf:580]: I agree with the suggestion

Comment (viccuad) [reply terraform/repository_policies.tf:580]: `validate-pvc-storageclass-policy`?
or
`pvc-storagelass-policy`? we can then have a series of `pvc-foo-policy`.

Comment (viccuad) [reply terraform/repository_policies.tf]: rebased with `persistentvolumeclaim-storageclass-policy`.

**Pull request merged**